test/unit/measure-loader/cql_loader_test.rb

```diff
@@ -119,19 +119,40 @@ def test_definition_with_same_name_as_a_library_definition
     end
   end
 
+  def test_source_data_criteria_creation
+    VCR.use_cassette('measure__source_data_criteria_creation', @vcr_options) do
+      measure_file = File.new File.join(@fixtures_path, 'CMS134v6.zip')
+      value_set_loader = Measures::VSACValueSetLoader.new(options: @vsac_options, ticket_granting_ticket: get_ticket_granting_ticket_using_env_vars)
+      loader = Measures::CqlLoader.new(measure_file, @measure_details, value_set_loader)
+      measures = loader.extract_measures
+      assert_equal 1, measures.length
+      measure = measures[0]
+
+      assert_equal measure.source_data_criteria.map(&:hqmfTitle), ["Encounter, Performed", "Procedure, Performed", "Patient Characteristic Sex", "Medication, Active", "Diagnosis", "Intervention, Order", "Intervention, Performed", "Patient Characteristic Race", "Patient Characteristic Payer", "Patient Characteristic Ethnicity", "Laboratory Test, Performed"]
```

Comment:
Fixes for Metrics/LineLength and Style/StringLiterals: 

```suggestion
      assert_equal measure.source_data_criteria.map(&:hqmfTitle), [
          'Encounter, Performed',
          'Procedure, Performed',
          'Patient Characteristic Sex',
          'Medication, Active',
          'Diagnosis',
          'Intervention, Order',
          'Intervention, Performed',
          'Patient Characteristic Race',
          'Patient Characteristic Payer',
          'Patient Characteristic Ethnicity',
          'Laboratory Test, Performed'
          ]
```